docs/sdks_03_core.md

```diff
@@ -27,83 +27,118 @@ yarn add @gnosis.pm/safe-core-sdk
 
 ## Getting Started
 
-A Safe account with three owners and threshold equal three will be used as the starting point for this example but any Safe configuration is valid.
+### 1. Set up the SDK using `Ethers` or `Web3`
+
+If the app integrating the SDK is using `Ethers` `v5`, create an instance of the `EthersAdapter`. `owner1` is the Ethereum account we are connecting and the one who will sign the transactions.
 
 ```js
 import { ethers } from 'ethers'
-import EthersSafe, { SafeTransactionDataPartial } from '@gnosis.pm/safe-core-sdk'
+import { EthersAdapter } from '@gnosis.pm/safe-core-sdk'
 
 const web3Provider = // ...
 const provider = new ethers.providers.Web3Provider(web3Provider)
-const signer1 = provider.getSigner(0)
-const signer2 = provider.getSigner(1)
-const signer3 = provider.getSigner(2)
+const owner1 = provider.getSigner(0)
+
+const ethAdapterOwner1 = new EthersAdapter({
+  ethers,
+  signer: owner1
+})
+```
+
+If the app integrating the SDK is using `Web3` `v1`, create an instance of the `Web3Adapter`.
+
+```js
+import Web3 from 'web3'
+import { Web3Adapter } from '@gnosis.pm/safe-core-sdk'
+
+const ethAdapterOwner1 = new Web3Adapter({
+  web3,
+  signerAddress: await owner1.getAddress()
+})
+```
+
+### 2. Deploy a new Safe
+
+To deploy a new Safe account instantiate the `SafeFactory` class and call the method `deploySafe` with the right params to configure the new Safe. This includes defining the list of owners and the threshold of the Safe. A Safe account with three owners and threshold equal three will be used as the starting point for this example but any Safe configuration is valid.
```

Comment:
I think in tech docs, number should be written as numbers and not text. Helps readability, i.e. instead of 
> A Safe account with three owners and threshold equal three will be used as the starting point for this example but any Safe configuration is valid.

write

A Safe account with 3 owners and threshold 3 will be used as the starting point for this example but any Safe configuration is valid.